src/app/serviceaccount/add-serviceaccount/template.html

```diff
@@ -29,20 +29,33 @@
           Name is <strong>required</strong>.
         </mat-error>
       </mat-form-field>
-      <mat-form-field fxFlex>
-        <mat-label>Group*</mat-label>
-        <mat-select id="km-add-serviceaccount-dialog-group-combobox"
-                    panelClass="km-add-dialog-dropdown"
-                    disableOptionCentering
-                    formControlName="group">
-          <mat-option value="editors">Editor</mat-option>
-          <mat-option value="projectmanagers">Project Manager</mat-option>
-          <mat-option value="viewers">Viewer</mat-option>
-        </mat-select>
-        <mat-error *ngIf="addServiceAccountForm.controls.group.hasError('required')">
-          Group is <strong>required</strong>.
-        </mat-error>
-      </mat-form-field>
+      <label class="km-radio-group-label">Select a Group</label>
+      <mat-radio-group formControlName="group"
+                       fxFlex
+                       fxLayout="column"
+                       class="km-radio-group">
+        <mat-radio-button value="projectmanagers"
+                          class="km-radio-button">
+          <div class="km-radio-button-title">Project Manager</div>
+          <div class="km-radio-button-description">
+            Project, members, and service accounts management, no access to clusters
```

Comment:
This might be a bit confusing. Why is the Project Manager not able to access clusters?
Is this from the "new service account" topic we talked about a few weeks ago?